### PR TITLE
bugs/fix-bottom-nav-bar-styles

### PR DESCRIPTION
**Pull request by bipin-shrestha**
Added some CSS styling the footer.

### CODE REVIEW

File: src/components/pages/WordCloud/CropCloud.js [hunk 1]
```diff
@@ -38,7 +38,6 @@ export default function CropCloud() {
           src={`data:image/png;base64, ${placeholderCloud.data}`}
         />
       </div>
- main
```
Comment (Mitch98k) [inline src/components/pages/WordCloud/CropCloud.js:41]: Could you explain the purpose of removing this line?

File: src/styles/less/GamemodeStyles.less [hunk 2]
```diff
@@ -45,14 +45,14 @@
...
-  margin-top: 2rem;
-}
+  .mission-explosion {
+    position: absolute;
+    // z-index: -1;
+  }
+  .mission-btn-txt {
+    z-index: 1;
+    margin-top: 2rem;
+  }
```
Comment (Mitch98k) [inline src/styles/less/GamemodeStyles.less:55]: I'm guessing you accidentally removed these lines and had to add them back in. Am I correct or was there some change you made that I failed to notice?

**Review by Mitch98k — APPROVED** [review]

File: src/components/pages/WordCloud/CropCloud.js (hunk 1 above)
Comment (bipin-shrestha) [reply src/components/pages/WordCloud/CropCloud.js:41]: "main" was appearing right in the middle of the page after the crop-cloud-container.

File: src/styles/less/GamemodeStyles.less (hunk 2 above)
Comment (bipin-shrestha) [reply src/styles/less/GamemodeStyles.less:55]: that's right.

**Comment by bipin-shrestha** [comment]
[CSS error on landing page after Log in.](https://trello.com/c/op0A7G20/745-css-error-on-landing-page-after-log-in)

**Review by PVigar88 — APPROVED** [review]
looks good to me

**Review by ashtilawat — APPROVED** [review]

**Pull request closed without merging**